http/server.go

```diff
@@ -47,14 +47,18 @@ func (t traceHandler) ServeHTTP(writer http.ResponseWriter, request *http.Reques
 	}
 	defer t.scope.Func().RemoteTrace(&ctx, parent, trace)(nil)
 
+	if cb, exists := trace.Get(present.SampledCBKey).(func(*monkit.Trace)); exists {
+		cb(trace)
+	}
+
 	s := monkit.SpanFromCtx(ctx)
 	s.Annotate("http.uri", request.RequestURI)
 
-	wrapped := &responseWriterObserver{w: writer}
+	wrapped, statusCode := Wrap(writer)
```

Comment:
I thought that statusCode is written by the handler, which is called in line 61

Reply:
Sorry, I don't understand this comment. What is your suggestion?

`statusCode` can be written by any other call on the writer. The goal here is to cache it, and annotate span with it at the end.

Reply:
My bad, I didn't realize that `Wrap` returns a function, my brain played the trick of seeing that it returned an int.